### PR TITLE
Add warning to input component

### PR DESCRIPTION
**Pull request by missx**
Add a warning prop to the input component.

## Risks

None

## Changes

image

### CODE REVIEW

File: src/components/Input/index.js [hunk 1]
```diff
@@ -60,6 +62,9 @@ export default class Input extends PureComponent {
         {error &&
           <span className='input-field__error'>{error}</span>
         }
+        {warning &&
+          <span className='input-field__error'>{warning}</span>
```
Comment (bernabe9) [inline src/components/Input/index.js]: maybe change the class name to `input-field__warning`?

Comment (bernabe9) [reply src/components/Input/index.js]: So the developers are able to override the styles outside the library with different classes

Comment (missx) [reply src/components/Input/index.js]: You're right!

**Comment by bernabe9** [comment]
@missx update your tests(`yarn test -u`) and push again to pass CircleCI

**Pull request merged**